packages/server/docker/docker-compose-staging.yml

```diff
@@ -6,14 +6,26 @@ services:
     restart: "always"
     volumes:
       - cargo-staging:/data/db
-  signlab:
+  cargo:
     image: hicsail/cargo:main
```

Comment:
could we add names to all the containers, otherwise it uses a default name in portainer and can get hard to read.

Reply:
Yep sounds good, added `container_name`